rules/use-backchannels-effectively/rule.md

```diff
@@ -21,75 +21,94 @@ created: 2020-08-31T23:13:15.000Z
+* **Post backchannels** – Debrief your manager or teammate how the call or situation was handled, outcomes, where it is at now, next steps, etc.
 
 In general, use Teams for private information, or SMS as a last resort. Treat sensitive information or scenarios with care to ensure a good outcome for all parties.
 
-**When something feels "off"** - Often, you might feel that a decision made by a colleague or manager is not quite right. In such cases, backchanneling can help clarify and resolve these situations. Here’s how you should handle it:
+### Using Teams chat to backchannel
 
-::: greybox  
+When using a private Teams Chat to backchannel a Teams Meeting, it's a good idea to name the chat at prefix with "**Backchannel - {{ MEETING NAME }}**".
+
+This makes it purpose of the chat obvious and reduces the risk of private messages being mistakenly sent through to the wider audience.
+
+### Let everyone know when something feels "off"
+
+Often, you might feel that a decision made by a colleague or manager is not quite right. In such cases, backchanneling can help clarify and resolve these situations. Here’s how you should handle it:
 
+::: greybox  
 In a meeting of 5 people, which includes the Product Owner, Solution Architect and 3 Software Engineers, the Software Engineers disagree with a decision made during the meeting. They start a backchannel to talk about it, without the Product Owner and Solution Architect involved, and reach a decision to not proceed.
 :::
-
 ::: bad
-
-**Figure: Bad example - Not all team members are involved in the backchannel**
+Figure: Bad example - Not all team members are involved in the backchannel
 :::
 
 ::: greybox
-
 3 Software Engineers disagree with a decision made in a meeting. They open a group chat with the Product Owner and Solution Architect soon after to discuss the decision.
 :::
 ::: good
-
-**Figure: Good example - All team members, including the Product Owner and Solution Architect, are involved in a group chat and are kept informed**
+Figure: Good example - All team members, including the Product Owner and Solution Architect, are involved in a group chat and are kept informed
 :::
 
-If the Software Engineers still disagree after the group chat, then they should make the matter official by sending a ['For the record'](http://www.ssw.com.au/for-the-record) email.
+#### Use "For the record" if disagreement persists
 
-*For example:  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"For the record… I understand you want microservices because of xxx\
- &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;However I think that is wrong because of yyy\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;I have CCed [3months@followupthen.com](mailto:3months@followupthen.com) to see if any of us have changed our mind"*
+If the Software Engineers still disagree after the group chat, then they should make the matter official by sending a ['For the record'](/for-the-record) email.
```

Comment:
Does this violate the principle of DRY?  We already have a rule on 'For the record'.  Won't this mean that in the future we need to update both templates?

Reply:
@leahy268 

From my perspective this is **not** a "for the record" template... but an example.

I personally don't mind removing the example and keeping just the link... but I thought the example was a requirement for this rule.

Reply:
Yes I think the template needs to be removed

Reply:
Hi @leahy268 

Done - removed email template to follow DRY

1. Please review again ;)